src/screens/Connections/Connections.tsx

```diff
@@ -71,6 +75,16 @@ const Connections = () => {
     }, 2000);
   }, []);
 
+  const onPress = () => {
+    if (modalRef && modalRef.current && modalRef.current.close) {
```

Comment:
Here we just want to make sure that modalRef.current is defined thats why we can check for only one condition 
```suggestion
    if (modalRef.current) {
```